animeshgarai09/felix-social-react
# Auth redux

This commit might be a little big and sorry for that.🙂 
I am using Redux toolkit with RTK Query for global state management and API requests.

Feature sto test in this PR:
- Sign in / Sign Up / Logout
- Verify the user if token is present ( Only with guest login)
- Visit profiles for logged in or other -(https://felix-social-react-git-auth-redux-animeshgarai09.vercel.app/profile/jamesbond007)
- Edit profile details
- Follow / Unfollow
- Restricted auth and require auth pages.
